firebase-firestore/src/testUtil/java/com/google/firebase/firestore/testutil/TestUtil.java

```diff
@@ -407,11 +416,16 @@ public static RemoteEvent addedRemoteEvent(
 
               @Override
               public QueryData getQueryDataForTarget(int targetId) {
-                return queryData(targetId, QueryPurpose.LISTEN, doc.getKey().toString());
+                return queryData(targetId, QueryPurpose.LISTEN, collectionPath.toString());
```

Comment:
Question: what's the reason for this change?

Reply:
The new test passes multiple docs to `addedRemoteEvent`. The test helper now uses the collection path of the first doc to represent the query.

FWIW I also believe the original implementation should have returned a collection path here.